client/src/partials/purchase/confirm/confirm.html

```diff
@@ -24,10 +24,10 @@
 
           <div class="alert alert-info po-alert">
             <strong>{{ 'PURCHASE.PURCHASE_TYPE' | translate }} : </strong>
-            <input type="radio" id="indirectPO" ng-model="session.is_direct" ng-value="false" ng-click="resetSelected()">
+            <input type="radio" id="indirectPO" ng-model="purchaseConfirmCtrl.session.is_direct" ng-value="false" ng-click="purchaseConfirmCtrl.resetSelected()">
```

Comment:
Because we are using Controller As syntax, we no longer need the object `session`.  Could you remove it?